src/ontology/mondo-edit.obo

```diff
@@ -107747,7 +107747,7 @@ relationship: disease_has_feature HP:0010762 ! Chordoma
 [Term]
 id: MONDO:0006146
 name: chondroid hamartoma
-def: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C42589&ns=NCI_Thesaurus" [EFO:1000175]
+def: "Https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=C42589&ns=NCI_Thesaurus" [EFO:1000175]
```

Comment:
I don't think we should have hyperlinks in the def field, I imagine this is a mistake